scene/2d/cpu_particles_2d.cpp

```diff
@@ -1561,6 +1563,7 @@ CPUParticles2D::CPUParticles2D() {
 	set_emitting(true);
 	set_amount(8);
 	set_use_local_coordinates(false);
+	set_seed(Math::rand());
```

Comment:
Calling `set_seed` instead of assigning `seed` directly will mean the seed generated in the editor will be saved, or should, this might not be desired

Reply:
It's marked with property hint none when fixed seed is off, so shouldnt be a problem